package.json

```diff
@@ -5,27 +5,21 @@
   "homepage": "https://mooniak.github.io/fontpad",
   "devDependencies": {
     "babel-eslint": "^7.1.1",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-plugin-transform-es2015-arrow-functions": "^6.22.0",
     "eslint": "^3.12.2",
     "eslint-config-airbnb": "^13.0.0",
     "gh-pages": "^0.12.0",
-    "react-scripts": "0.8.4",
-    "webpack": "^1.12.12"
```

Comment:
Do we need to include webpack here and the webpack-dev-server below? react-scripts have them as dependencies.